.github/workflows/deploy.yaml

```diff
@@ -17,19 +17,18 @@ on:
         - staging
         - production
         default: 'integration'
-  push:
-    branches:
-      - main
-    paths-ignore:
-      - "Jenkinsfile"
-      - ".git**"
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
```

Comment:
You might want to keep this as on push - until we've got CI set up properly

Reply:
ok